src/nodedef.cpp

```diff
@@ -676,12 +676,12 @@ void ContentFeatures::updateTextures(ITextureSource *tsrc, IShaderSource *shdsrc
 
 	bool is_liquid = false;
 
-	u8 material_type = (alpha == 255) ?
-		TILE_MATERIAL_BASIC : TILE_MATERIAL_ALPHA;
```

Comment:
As i understand it ..
Previously tile material for most drawtypes was decided by 'alpha'.
Now, tile material alpha is set as the default but changed to material basic for normal drawtype.

However i now see that more consistent behaviour would happen if these 2 lines were unchanged, material tye would still be forced to 'basic' for normal drawtype.